src/base/rss/rss_autodownloadrule.h

```diff
@@ -79,8 +83,8 @@ namespace RSS
         void setSavePath(const QString &savePath);
         TriStateBool addPaused() const;
         void setAddPaused(TriStateBool addPaused);
-        TriStateBool createSubfolder() const;
-        void setCreateSubfolder(TriStateBool createSubfolder);
+        boost::optional<BitTorrent::TorrentContentLayout> torrentContentLayout() const;
+        void setTorrentContentLayout(boost::optional<BitTorrent::TorrentContentLayout> contentLayout);
```

Comment:
Seeing this, I think we should define the following:
```c++
// in torrentcontentlayout.h
using BitTorrent::TorrentContentLayout = boost::optional<BitTorrent::TorrentContentLayoutEnum>;  // names can be improved...

// using it, this looks cleaner IMO
BitTorrent::TorrentContentLayout torrentContentLayout() const;
void setTorrentContentLayout(BitTorrent::TorrentContentLayout contentLayout);
```

Reply:
>Seeing this, I think we should define the following:

I'd like to have things explicit.